guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
```

Comment:
This feels run-on and inorganic, `of autocomplete` doesn't seem to be proper English, `command` should probably be plural or replaced with `options.`